src/components/AnnotationPopup/AnnotationPopup.js

```diff
@@ -183,6 +184,7 @@ class AnnotationPopup extends React.PureComponent {
 }
 
 const mapStateToProps = state => ({
+  test: selectors.getActiveToolName(state),
```

Comment:
This prop doesn't seem to be used in this component, can we remove this line if this is indeed not being used?